data.json

```diff
@@ -0,0 +1 @@
+{"names": {"132551667085344769": "dam4rusxp", "413068385962819584": "galaali"}}
```

Comment:
die datei sollte nicht committed werden

Reply:
warum ist es dann nicht in gitignore?

Reply:
als standard DATA_DIR wird in der Dockerfile data/ gesetzt, das kann man auch ins env file schreiben. ist nicht ganz optimal, kann man nochmal schöner machen